tests/transactional/host_config.pm

```diff
@@ -1,24 +1,31 @@
 # SUSE's openQA tests
 #
-# Copyright 2020-2023 SUSE LLC
+# Copyright 2020-2024 SUSE LLC
 # SPDX-License-Identifier: FSFAP
 #
 # Package: transactional-update
-# Summary: Host configuration operations (e.g. disable grub timeout,
-#              kernel params, etc)
+# Summary: Host configuration operations
+# * Configure virtio-terminal
+# * Configure bootloader (disable timeout, extra kernel parameters)
+# * Install SUSE CA certificates, if applicable
 #
 # Maintainer: QE-C team <qa-c@suse.de>
 
 use Mojo::Base qw(consoletest);
 use testapi;
 use transactional qw(process_reboot);
 use bootloader_setup qw(change_grub_config);
-use utils qw(ensure_ca_certificates_suse_installed zypper_call);
+use utils qw(ensure_ca_certificates_suse_installed zypper_call ensure_serialdev_permissions);
 use version_utils qw(is_bootloader_grub2 is_bootloader_sdboot);
+use serial_terminal qw(select_serial_terminal prepare_serial_console);
 
 sub run {
     select_console 'root-console';
 
+    ensure_serialdev_permissions;
+    prepare_serial_console;
+    select_serial_terminal;
```

Comment:
**TL;DR: feel free to merge**, although I'd prefer less duplicity (DRY). That's the only thing I'm trying to point out, but feel free to ignore.

`prepare_serial_console()` was supposed to be the only function (TM) which should be called to make sure serial terminal is possible to be used. But instead I should have created it as a single module, which would be loaded everywhere. But even now, `prepare_serial_console;` is in several modules, maybe tests/jeos/firstrun.pm or tests/console/system_prepare.pm could have been used instead of adding function to yet another module.

I did not know about `ensure_serialdev_permissions()`. If this is needed for virtio terminal (and not for dmesg) then IMHO it should have been in `prepare_serial_console()`.

Reply:
We need to do this in a second PR then. I was taking the template from e.g. https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/44d247ec466ad36991a4f1c3148117851912345c/tests/console/system_prepare.pm#L34 where the following procedure is used:

```
ensure_serialdev_permissions;
prepare_serial_console;
```

I think we can merge the two together, but given that this is not yet done in other places, I would like to keep this one as it is and do this small refactoring later.